styles/bilibili/catppuccin.user.less

```
@@ -0,0 +1,724 @@
+/* ==UserStyle==
+@name Bilibili Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/bilibili
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/bilibili
+@version 0.1.0
```

```suggestion
@version 2000.01.01
```

"You may notice that userstyles contain a line in the format of @version 2000.01.01 in the UserCSS metadata section at the top. This line should not be edited by hand."
 https://userstyles.catppuccin.com/contributing/#:~:text=You%20may%20notice%20that%20userstyles%20contain%20a%20line%20in%20the%20format%20of%20%40version%202000.01.01%20in%20the%20UserCSS%20metadata%20section%20at%20the%20top.%20This%20line%20should%20not%20be%20edited%20by%20hand.